Formula/c2ffi.rb

```diff
@@ -0,0 +1,217 @@
+class C2ffi < Formula
+  desc "Clang-based FFI wrapper generator"
+  homepage "https://github.com/rpav/c2ffi"
+  url "https://github.com/rpav/c2ffi/archive/llvm-11.0.0.tar.gz"
+  version "llvm-11.0.0"
+  sha256 "7bc32f294c4d9e7e6ee3e04d45b1b8c04ae51872dbd90e8458c82c4cb98208e5"
+  license "LGPL-2.1-only"
+  head "https://github.com/rpav/c2ffi.git", branch: "llvm-11.0.0"
+
+  depends_on "cmake" => :build
+  depends_on "llvm@11"
+  depends_on "qt@5"
+
+  on_linux do
+    depends_on "gcc"
+  end
+
+  fails_with gcc: "5" # needs C++17
+
+  def install
+    ENV["LDFLAGS"] = "-L/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/lib"
+    system(
+      "cmake", "-S", ".", "-B", "build",
+      *std_cmake_args,
+      "-DBUILD_CONFIG=Release",
+      "-DCMAKE_C_COMPILER=#{Formula["llvm@11"].opt_bin/"clang"}",
+      "-DCMAKE_CXX_COMPILER=#{Formula["llvm@11"].opt_bin/"clang++"}"
```

Comment:
As an aside, for when this formula is ready for Homebrew/core: we usually avoid setting `CMAKE_C[XX]_COMPILER` directly like this, because this evades our compiler shims. It would be good to find a way to not have to do this. Using the compiler shims should also avoid the need for setting `LDFLAGS` as you do above, I think.

Reply:
This tool must be pinned to a specific version of LLVM, and this was the only way I knew how to achieve that. I don't know enough about C compilers or CMake to have any other ideas...